ansible_base/rbac/models.py

```diff
@@ -705,7 +715,7 @@ class Meta(RoleEvaluationMeta):
     role = models.ForeignKey(
         ObjectRole, null=False, on_delete=models.CASCADE, related_name='permission_partials', help_text=_("The object role that grants this form of permission")
     )
-    object_id = models.PositiveIntegerField(null=False)
+    object_id = models.PositiveIntegerField(null=False, help_text=_("The object ID this role evaluation will be applied to."))
```

Comment:
Grammar seems a little backwards. This is a statement of role-object-permission.

> The id of the object that the related role gives the related permission to